context/app/static/js/components/Detail/provenance/ProvTableTile/ProvTableTile.jsx

```diff
@@ -15,7 +15,7 @@ function ProvTableTile(props) {
   // mapped fields are not included in ancestor object
   const entityData = useEntityData(uuid, elasticsearchEndpoint, nexusToken);
 
-  const allDescendantCounts = useDescendantCounts(entityData, ['Sample', 'Dataset']);
+  const allDescendantCounts = useDescendantCounts(entityData, ['Sample', 'Dataset', 'Support']);
```

Comment:
We'll have support entities in the descendants metadata, right?

Reply:
Not sure about this... Just going by memory, I would think it's just at the top of the entity json, not in the nested structures. I'll file a follow-up issue.